app/views/organisations/_show_organisation.html.erb

```diff
@@ -49,6 +49,30 @@
   <% end %>
 </div>
 
+<% if @people.ministers.any? && !@organisation.is_no_10?%>
+  <div class="organisations__section organisations__brand-border-top brand--<%= @organisation.brand %> brand__border-color">
```

Comment:
What's the `organisations__brand-border-top` class? It doesn't seem to do anything?

Reply:
It adds the blue/brand-colour border to the top of each section on the page.